webapp/pages/settings.vue

```diff
@@ -39,6 +39,10 @@ export default {
           name: this.$t('settings.social-media.name'),
           path: `/settings/my-social-media`,
         },
+        {
+          name: this.$t('settings.myGroups'),
+          path: `/group/my-groups`,
+        },
```

Comment:
```suggestion
        {
          name: this.$t('settings.myGroups'),
          path: `/my-groups`,
        },
```

I remember that we liked to use this suggested route for the list of my groups.

Reply:
done